javalib/src/main/scala/java/util/Throwables.scala

```diff
@@ -42,16 +42,15 @@ class IllegalFormatCodePointException(private val c: Int) extends IllegalFormatE
   override def getMessage(): String = s"Code point = $c"
 }
 
-class IllegalFormatConversionException private(private val c: Char) extends IllegalFormatException {
-  private var arg: Class[_] = null
-  def this(c: Char, arg: Class[_]) = {
-    this(c)
-    if (arg == null)
-      throw new NullPointerException()
-    this.arg = arg
-  }
+class IllegalFormatConversionException(c: Char, arg: Class[_])
+    extends IllegalFormatException {
+
+  if (arg == null)
+    throw new NullPointerException()
+
```

Comment:
So just to clarify: `toString` will still crash here if `arg` is null, no? The difference is just that the throw point comes after the assignment to arg.

Reply:
Yes, if `arg` is `null`, `toString` will crash, but so would the constructor. So that's not a big deal. The important part is that `toString` does not crash when called too early, but in a situation where it would not crash if were called later.